README.md

```diff
@@ -119,11 +125,14 @@ beyond the LTS release.
   <td>2018-04-01</td>
 </tr>
 <tr>
-  <td>v.Next</td>
+  <td>v6.x.x</td>
   <td>2016-10-01</td>
   <td>2018-04-01</td>
   <td>2019-04-01</td>
 </tr>
 </table>
 
-<p><img src="strawmanschedule.png" alt="Strawman LTS Schedule"/></p>
+<p><img src="https://camo.githubusercontent.com/675f052419c56a583c19644d4da30cc0ff4e02bb/68747470733a2f2f636c6475702e636f6d2f6c4d48746245334e72782d3330303078333030302e706e67" alt="Strawman LTS Schedule"/></p>
+
+(note: the diagram above shows v0.12 maintenance expiring in April 2017. That
+date has been changed to December 31st, 2016.)
```

Comment:
Why does this PR update the actual in-tree LTS Schedule with an out-of-tree Draft schedule, that has one date invalid?